README.md

```diff
@@ -20,21 +20,50 @@ This bookdown book is a *work in progress*. We'll update this `README` and the r
     
 ## Required packages
 
+
+### Use pak to install any missing packages
+
+Here's one way to install the needed packages (only the ones that you don't already have) using the [pak package](https://pak.r-lib.org/index.html).
+
 <!--TODO: Change pkg_list to not be static, maybe use renv::dependencies(path = "DESCRIPTION")?-->
 
-```{r eval = FALSE}
+```r
 pkg_list <- c("bookdown", "devtools", "dichromat", "DT", "fs", "gapminder",
               "gender", "geonames", "git2r", "glue", "gridExtra",  "htmltools",
               "httr", "knitr", "RColorBrewer", "rebird", "rmarkdown", "rplos", 
               "rvest", "testthat", "tidyverse", "usethis", "viridis", "xfun", 
               "xml2", "ropensci/genderdata", "rstudio/gt", "rstudio/renv@46f1123")
 ```
 
-Here's one way to install the needed packages (only the ones that you don't already have) using the [`pak` package](https://pak.r-lib.org/index.html).
 
-```{r eval = FALSE}
+```r
 # install.packages("pak")
 pak::pkg_install(pkg_list)
 ```
 
-<!--TODO: Add a second option using the `renv` package.-->
\ No newline at end of file
+### Use renv to recreate our project library
+
+Another option is to use the [renv package](https://rstudio.github.io/renv/index.html) to replicate our exact project library. renv is a package that uses a [snapshot and restore](https://environments.rstudio.com/snapshot.html) strategy to create **r**eproducible **env**nvironments for R projects. renv will create a private, project-specific library that is separate from your personal library of packages. This would be a good option if, for example, you have another project that relies on a specific version of a package and you don't want to mess with it by upgrading, downgrading, etc.
+
+If you want to learn more about what renv is doing behind the scenes checkout [renv's main site](https://rstudio.github.io/renv/index.html).
+
+*Note: The renv package is still in the development stage so these instructions may change over time!*
+
+Once you have a local copy of this project (either via [fork and clone](https://happygitwithr.com/fork-and-clone.html) or by downloading this repository as a ZIP file), follow these steps:
+
+1. Install the development version of the renv package -- specifically, the same version we used to take a "snapshot" of our project library.
+   
+    ```r
+    if (!requireNamespace("remotes"))
+      install.packages("remotes")
+  
+    remotes::install_github("rstudio/renv@e0a0c13")
```

Comment:
So it's really necessary to be installed from this specific SHA? If not, we should relax this. If it is, we need a comment close by that explains why it's important, so I / Education Team know when to remove this.

Reply:
Also, we have a weak preference for devtools in these settings:, e.g. `devtools::install_github()`, instead of remotes.

https://r-pkgs.org/setup.html#setup-usage

I would also just put the `install.package("devtools")` command in a comment, as you do elsewhere here, e.g. for pak.

Reply:
I think so. I've updated my local installation of `renv` twice over the past ~two months and both times I had to update these instructions because something had changed.

Reply:
Well, I still think we should "chase" renv. It's very close to a CRAN release, so I think we should all be using bleeding edge and adapting these instructions as necessary.